src/05-led-roulette/README.md

```diff
@@ -46,4 +42,17 @@ as well. This directory contains a Cargo configuration file (`.cargo/config`) th
 linking process to tailor the memory layout of the program to the requirements of the target device.
 This modified linking process is a requirement of the `cortex-m-rt` crate.
 
```

Comment:
Might be worth it explaining a bit about cargo-embed/probe-rs here, including that it is much better than traditional openocd and so on

Reply:
I'm not quite sure whether saying cargo-embed is better than X is a good thing to do here, people interested in the tooling will be able to find its website etc. quite easily but surely there are things OpenOCD does better than cargo-embed, such as for example the GDB server.